subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/maven/PomProjectInitDescriptor.java

```diff
@@ -73,16 +82,35 @@ public String getSourceBuildDescription() {
         return "Maven";
     }
 
+    @Override
+    public void configureClasspath(ProjectInternal.DetachedResolver detachedResolver, ObjectFactory objects) {
+        DependencyHandler dependencies = detachedResolver.getDependencies();
+        Configuration config = detachedResolver.getConfigurations().detachedConfiguration(
+            dependencies.create("org.apache.maven:maven-core:3.6.3"),
```

Comment:
Would be nice to be avoid having those dependencies hardcoded in the sources, but that's good for now.